keras_contrib/applications/densenet.py

```diff
@@ -43,7 +43,7 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16, nb_layers
         optionally loading weights pre-trained
```

Comment:
Add activation parameter to DenseNet as well. Not only is it useful in regular DenseNet for the two class case, it will also be very useful when combined with Atrous DenseNet in #46.